### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by cloutiertyler**
# Description of Changes

This pull request includes updates to the `README.md` file to clarify the status of various language SDKs and modules for SpacetimeDB. The most important changes include updating the status of the TypeScript and Python SDKs and modules from "coming soon" to "planned."

Updates to SDK and module status:

* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L175-R176): Changed the status of TypeScript and Python modules from "coming soon" to "planned."
* [`README.md`](diffhunk://#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5L185-R185): Changed the status of the Python SDK from a link to "planned."

### CODE REVIEW

**Review by bfops — APPROVED** [review]
Excellent :raised_hands:

File: README.md [hunk 1]
```diff
@@ -182,7 +182,7 @@ You can write SpacetimeDB modules in a bunch of popular languages, with many mor
 - [Rust](https://spacetimedb.com/docs/sdks/rust/quickstart)
 - [C#](https://spacetimedb.com/docs/sdks/c-sharp/quickstart)
 - [Typescript](https://spacetimedb.com/docs/sdks/typescript/quickstart)
-- [Python](https://spacetimedb.com/docs/sdks/python/quickstart)
+- Python (planned)
```
Comment (bfops) [inline README.md:185]: good catch.. I wonder what I searched for / why I missed this..

Comment (cloutiertyler) [reply README.md:185]: I think you correctly made the change to the docs, but it's also listed here.

Comment (bfops) [reply README.md:185]: oh yeah this is a different repo :upside_down_face:

**Pull request merged**